content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md

```diff
@@ -65,21 +65,18 @@ With the above credentials ready, you can use the user `project-regular` to crea
    ```groovy
    pipeline {
      agent {
-       node {
-         label 'maven'
-       }
-   
+       label 'go'
      }
      
      environment {
        REGISTRY = 'docker.io'
        // Docker Hub username
        DOCKERHUB_USERNAME = 'Your Docker Hub username'
        APP_NAME = 'devops-go-sample'
-       // ‘dockerhub-go’ is the Docker Hub credentials ID you created on the KubeSphere console
-       DOCKERHUB_CREDENTIAL = credentials('dockerhub-go')
+       // ‘dockerhub’ is the Docker Hub credentials ID you created on the KubeSphere console
```

Comment:
I'd prefer to change `‘` to `'` or `"`.

Reply:
why?

Reply:
`‘` sign is so weird in documentation written in English.

![image](https://user-images.githubusercontent.com/16865714/141264843-9406ccc1-dca0-4600-9075-df73d39467a9.png)

Reply:
Got you.